src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -565,18 +568,20 @@ void style_transfer::perform_predict(gl_sarray data, gl_sframe_writer& result,
       {{"st_num_styles", st_num_styles}, {"st_training", st_train}},
       weight_params);
 
-  // Style Printer
-  size_t idx = 0;
-  table_printer table(
-        { {"Images Processed", 0}, {"Elapsed Time", 0}, {"Percent Complete", 0} }, 0);
-  table.print_header();
-
   // looping through all of the data
   data_iter->reset();
   ASSERT_EQ(batch_size, 1);
   std::vector<st_example> batch = data_iter->next_batch(batch_size);
   int row_idx = 0;
 
+  // Style Printer
+  // record time of process
```

Comment:
move the recording start point to be closer to the critical code blocks. might be only a few milliseconds more accurate.